--- a/install/make.inc.in
+++ b/install/make.inc.in
@@ -198,4 +198,6 @@
 WGET = @wget@
 
 # Install directory - "make install" copies *.x executables there
-PREFIX = @prefix@
+prefix = @prefix@
+exec_prefix = @exec_prefix@
+bindir = @bindir@
--- a/Makefile
+++ b/Makefile
@@ -297,10 +297,10 @@
 #############################################################
 
 install : 
-	mkdir -p $(PREFIX)/bin ; \
+	mkdir -p $(DESTDIR)$(bindir) ; \
 	for x in `find * ! -path "test-suite/*" -name *.x -type f` ; do \
-		cp -v $$x $(PREFIX)/bin/ ; done
-	@echo -e '\nQuantum ESPRESSO binaries are installed in $(PREFIX)/bin\n'
+		cp -v $$x $(DESTDIR)$(bindir)/ ; done
+	@echo -e '\nQuantum ESPRESSO binaries are installed in $(DESTDIR)$(bindir)\n'
 
 #########################################################
 # Run test-suite for numerical regression testing
