--- ./Modules/read_pseudo.f90.orig	2021-01-06 21:12:10.962520449 +0100
+++ ./Modules/read_pseudo.f90	2021-01-06 21:15:21.523514195 +0100
@@ -120,6 +120,18 @@
         INQUIRE ( file = file_pseudo, EXIST = exst) 
         IF (exst) ios = 0
         CALL mp_sum (ios,intra_image_comm)
+        !CALL errore('readpp', 'file '//TRIM(file_pseudo)//' not found',ABS(ios))
+     END IF
+     !
+     ! file not found? no panic (yet): try the system location pseudo_dir
+     ! '/usr/share/espresso/pseudo/'
+     !
+     IF ( ios /= 0 ) THEN
+	pseudo_dir = '/usr/share/espresso/pseudo/'
+        file_pseudo = TRIM (pseudo_dir) // TRIM (psfile(nt))
+        INQUIRE ( file = file_pseudo, EXIST = exst) 
+        IF (exst) ios = 0
+        CALL mp_sum (ios,intra_image_comm)
         CALL errore('readpp', 'file '//TRIM(file_pseudo)//' not found',ABS(ios))
      END IF
      !
