Index: espresso/EPW/src/Makefile
===================================================================
--- espresso.orig/EPW/src/Makefile
+++ espresso/EPW/src/Makefile
@@ -11,8 +11,7 @@ MODFLAGS= $(BASEMOD_FLAGS) \
           $(MOD_FLAG)../../PW/src \
           $(MOD_FLAG)../../PHonon/PH \
           $(MOD_FLAG)../../LR_Modules \
-          $(MOD_FLAG)../../wannier90-3.1.0/src/obj \
-          $(MOD_FLAG)../../dft-d3
+          $(MOD_FLAG)../../dft-d3 -I/usr/include
 
 #
 # The above is in alphabetical order except when order matter during compilation
@@ -77,7 +76,7 @@ EPWOBJS += ephblochkq.o wfc_elec.o test_
 
 PHOBJS = ../../PHonon/PH/libph.a
 PWOBJS = ../../PW/src/libpw.a
-W90LIB = ../../wannier90-3.1.0/libwannier.a 
+W90LIB = -lwannier
 LRMODS = ../../LR_Modules/liblrmod.a
 PWOBJS = ../../PW/src/libpw.a 
 QEMODS = ../../Modules/libqemod.a ../../KS_Solvers/libks_solvers.a \
@@ -86,13 +85,13 @@ LIBOBJS =../../LAXlib/libqela.a ../../Ut
 
 TLDEPS= pw ph
 
-all :   pw ph wannier libepw.a epw.x 
+all :   pw ph libepw.a epw.x 
 
 libepw.a : $(EPWOBJS)
 	$(AR) $(ARFLAGS) $@ $?
 	$(RANLIB) $@
 
-epw.x : epw.o libepw.a $(PHOBJS) $(LRMODS) $(PWOBJS) $(W90LIB) $(QEMODS) $(LIBOBJS)
+epw.x : epw.o libepw.a $(PHOBJS) $(LRMODS) $(PWOBJS) $(QEMODS) $(LIBOBJS)
 	$(LD) $(LDFLAGS) -o $@ \
 	epw.o  libepw.a	$(PHOBJS) $(LRMODS) $(W90LIB) $(PWOBJS) $(QEMODS) $(LIBOBJS) $(QELIBS) 
 	- ( cd ../bin ; ln -fs ../src/epw.x . )
@@ -103,9 +102,6 @@ pw :
 ph : 
 	cd ../../ ; make ph
 
-wannier :
-	cd ../../ ; make w90 ; cd wannier90-3.1.0/ ; make lib
-
 ld1 :
 	cd ../../ ; make ld1
 
